scripts/check_coverage.ps1

```diff
@@ -0,0 +1,8 @@
+$currentPath = Get-Location
```

Comment:
Are these powershell files for compatability with windows?

Reply:
Yes, exactly. I also wrote some code under windows. I am not sure if we should keep it or not. Btw. it is also possible to run powershell under linux and mac. Could be also possible to use this plattform indepentend.